analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeAnalysisContextExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2022 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace SonarAnalyzer.Extensions
+{
+    internal static class SyntaxNodeAnalysisContextExtensions
+    {
+        /// <summary>
+        /// Roslyn invokes the analzyer twice for positional records.
+        /// The first invocation is for the class declaration and the second for the ctor represented by the positional parameter list.
+        /// </summary>
+        /// <returns>
+        /// Returns <see langword="true"/> for the invocation on the class declaration and
+        /// <see langword="false"/> for the ctor invocation.
```

Comment:
Can be on a single line. Doesn't violate S103

Reply:
Done in 8baf21114883b150a33afa5b80e6a2d742192673